### ISSUE TITLE
Spanish translation (I already did it)

### ISSUE THREAD

**Issue by TheLegendofSaram** [gliscowo/interactic#28]
Here is the file, sorry to use the issue tab, i couldn't find the suggestion tab
[es_mx.zip](https://github.com/gliscowo/interactic/files/14463911/es_mx.zip)

**Comment by solonovamax** [comment]
> Here is the file, sorry to use the issue tab, i couldn't find the suggestion tab [es_mx.zip](https://github.com/gliscowo/interactic/files/14463911/es_mx.zip)

you can fork the repo and open a PR, or if you'd like I could do that for you.

to fork it, follow this guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo
and then to add the file, follow this guide: https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository
and then to create a PR, follow this guide: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

upload the file in the `src/main/resources/assets/interactic/lang` directory, as `es_ms.json` (select "commit directly to branch")

**Comment by TheLegendofSaram** [comment]
Yeah sorry that was when I first started using github, i think i alredy PR the translation from another repo

**Comment by TheLegendofSaram** [comment]
Thanks for the tutorial tho!!
